{"timestamp":"2025-08-19T19:11:31.431726","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-19T19:11:31.438034","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/dag_with_postgres_operator.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-19T19:11:32.210372","level":"info","event":"Executing: CREATE SCHEMA IF NOT EXISTS my_tests;","logger":"airflow.task.operators.airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator"}
{"timestamp":"2025-08-19T19:11:32.267091","level":"info","event":"Connection Retrieved 'DBeaverConn'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-19T19:11:32.340548","level":"info","event":"Connection Retrieved 'DBeaverConn'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-19T19:11:32.446134","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OperationalError","exc_value":"connection to server at \"host.docker.internal\" (192.168.65.254), port 5433 failed: FATAL:  database \"CoreLogic\" does not exist\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":877,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1164,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py","lineno":317,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":767,"name":"run"},{"filename":"/usr/local/lib/python3.12/contextlib.py","lineno":137,"name":"__enter__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":874,"name":"_create_autocommit_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py","lineno":177,"name":"get_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py","lineno":122,"name":"connect"}],"is_group":false,"exceptions":[]}]}
